app/controllers/user_ratings_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UserRatingsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[new]
+  def new
+    @user_rating = UserRating.new
+    @user = User.find(params[:user_id])
```

Comment:
```suggestion
```

Remove this line because it is no longer nested in users and this param won't be available